docs/api/v1/openapi.yml

```
@@ -1057,6 +1067,128 @@ paths:
+          schema:
+            type: string
+      requestBody:
+        description: Payload containing an array of one or more users to add to the project.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                relations:
+                  type: array
+                  description: A list of userIds and roles to add to the project.
+                  items:
+                    type: object
+                    properties:
+                      userId:
+                        type: string
+                        description: The unique identifier of the user.
+                        example: 91765f0d-3b29-45df-adb9-35b23937eb92
+                      role:
+                        type: string
+                        description: The role assigned to the user in the project.
+                        example: project:viewer
+                    required:
+                      - userId
+                      - role
+              required:
+                - relations
+      responses:
+        '201':
+          description: Operation successful.
+        '401':
+          $ref: '#/components/responses/unauthorized'
+        '403':
+          $ref: '#/components/responses/forbidden'
+        '404':
+          $ref: '#/components/responses/notFound'
+  /projects/{projectId}/users/{userId}:
+    delete:
+      x-eov-operation-id: deleteUserFromProject
+      x-eov-operation-handler: v1/handlers/projects/projects.handler
+      tags:
+        - Projects
+      summary: Delete a user from a project
+      description: Delete a user from a project from your instance.
+      parameters:
+        - name: projectId
+          in: path
+          description: The ID of the project.
+          required: true
+          schema:
+            type: string
+        - name: userId
+          in: path
+          description: The ID of the user.
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Operation successful.
+        '401':
+          $ref: '#/components/responses/unauthorized'
+        '403':
+          $ref: '#/components/responses/forbidden'
+        '404':
+          $ref: '#/components/responses/notFound'
+    patch:
+      x-eov-operation-id: changeUserRoleInProject
+      x-eov-operation-handler: v1/handlers/projects/projects.handler
+      tags:
+        - Projects
+      summary: Change a user's role in a project
+      description: Change a user's role in a project.
+      parameters:
+        - name: projectId
+          in: path
+          description: The ID of the project.
+          required: true
+          schema:
+            type: string
+        - name: userId
+          in: path
+          description: The ID of the user.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Updated project object.
```

```suggestion
        description: Payload containing the new role to assign to the project user.
```

Maybe? Just trying to match the style used in the `POST` action.